src/stactools/noaa_nclimgrid/stac.py

```diff
@@ -187,11 +187,15 @@ def create_items(
     return (items, created_cogs)
 
 
-def create_collection(frequency: Frequency, nc_assets: bool = False) -> Collection:
+def create_collection(
+    frequency: Frequency, prelim: bool = False, nc_assets: bool = False
+) -> Collection:
```

Comment:
How much extra work would it be to change `Frequency` to something like this (names can be changed, this is just a prototype):

```python
class CollectionType(Enum):
    Monthly
    DailyPrelim
    DailyScaled
```

This would keep the number of arguments in this function to two, and remove the ambigous case where `frequency == Monthly` and `prelim == True`.

This is a minor issue and if it's a decent amount of refactoring, not worth doing since this dataset is almost buttoned up.

Reply:
Good idea. I think it's worth it.